okhttp/src/main/java/okhttp3/PingPayloadProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I assume this should assign the Copyright (C) 2014 Square, Inc. as part of the CLA process.